swift/apple/FirezoneKit/Sources/FirezoneKit/Helpers/Log.swift

```diff
@@ -127,12 +139,6 @@ private final class LogWriter {
 
     jsonData.append(newLineData)
 
-    workQueue.async {
-      do {
-        try jsonData.write(to: self.logFileURL, options: .atomic)
```

Comment:
So it's this `write` function that was truncating instead of appending?

Reply:
Yeah, it writes the whole blob instead of appending.